runtime/bundles/org.eclipse.e4.core.di.extensions/META-INF/MANIFEST.MF

```diff
@@ -9,5 +9,6 @@ Bundle-RequiredExecutionEnvironment: JavaSE-17
 Export-Package: org.eclipse.e4.core.di.extensions;version="0.16.0"
 Bundle-Localization: fragment
 Import-Package: jakarta.inject;version="[2.0.0,3.0.0)",
- javax.inject;version="[1.0.0,2.0.0)";resolution:=optional
+ javax.inject;version="[1.0.0,2.0.0)";resolution:=optional,
+ org.osgi.framework;version="[1.0.0,2.0.0)"
```

Comment:
In order to not loose this dependency on the next dependency clean-up I suggest you replace the fully qualified class-names in `org.eclipse.e4.core.di.extensions.OSGiBundle` by regularly imported classes.
Then removing it would lead to a compiler error.

Reply:
Makes sense, thanks! Fixed with 3681b9b.